cue/context.go

```diff
@@ -143,6 +143,7 @@ func (c *Context) BuildInstances(instances []*build.Instance) ([]Value, error) {
 		v, err := c.runtime().Build(nil, b)
 		if err != nil {
 			errs = errors.Append(errs, err)
+			continue
```

Comment:
The only semantics that makes sense is to append an error value, or to, in case of any error, return no value at all. Just skipping an erroneous value like here will cause a misalignment in the values returned w.r.t. to the inputs.

Because an error Value can be useful, and to be more compatible with the deprecated `cue.Build`, I would suggest adding a Value representing the error, rather than skipping it. So add `c.makeError(err)` before continuing.

Reply:
LMK if this is what you had in mind!